external-lb.go

```diff
@@ -77,8 +76,8 @@ func updateExistingConfigs(metadataConfigs map[string]model.LBConfig, providerCo
 			mLBConfig := metadataConfigs[key]
 			pLBConfig := providerConfigs[key]
 			var update bool
-			//check that the targetName and targets match
-			if mLBConfig.LBTargetName == pLBConfig.LBTargetName {
+			//check that the targetPoolName and targets match
+			if mLBConfig.LBTargetPoolName == pLBConfig.LBTargetPoolName {
```

Comment:
@alena1108 I re-submitted the PR fixing following cases:
Fixed following usecase:
usecase 1)
Service s1 <--> vip1
And also Service s2 <--> vip1
Then Service s2 is skipped and not configured on f5

usecase 2)
Service s1 <--> vip1
Then user removed vip from s1 and then puts Service s2 <--> vip1
Then we remove the f5 config for s1 and add the config for s2.